dts/nxp/s32/S32Z27-BGA594-pinctrl.h

```diff
@@ -612,6 +612,9 @@
 #define PA11_DSPI_10_PCS0_O             NXP_S32_PINMUX(0, 0, 11, 1, 0, 0)
 #define PA12_DSPI_10_PCS1               NXP_S32_PINMUX(0, 0, 12, 1, 0, 0)
 #define PB6_DSPI_10_PCS2                NXP_S32_PINMUX(0, 0, 22, 6, 0, 0)
+#define DSPI_10_SIN                     NXP_S32_PINMUX(0, 0, 500, 0, 0, 0)
```

Comment:
why there are no prefix for the port/pin on these three pinmuxes?

Reply:
those pin belongs to LVDS pins and quite special, 
I see that we named the macros by "port" "_" "function"
however, I see in the IOMUX sheet the port is same as function :). Should we keep the same which i implemented?

Reply:
I would suggest to use to use `LVDS_` as prefix for the voltage differential pads, to indicate that they are not targetting single-ended GPIO pads that belong to a port/pin pair

Reply:
I updated it, thank